bin/main.rb

```diff
@@ -1,9 +1,40 @@
-class Game
-    def initialize
-        p 'Player 1, enter your name: '
-        @player1 = Player.new(gets.chomp, 'X')
-        p 'Player 2 enter your name: '
-        @player2 = Player.new(gets.chomp, 'O')
-        @board = Board.new
-    end
-end
\ No newline at end of file
+class Main
+  def initialize
+    @player1 = Player.new('X')
```

Comment:
The player class takes to arguments when initializing it so you have to do something like `@player1 = Player.new(gets.chomp, 'X')`